src/main/java/stirling/software/SPDF/config/security/SecurityConfiguration.java

```diff
@@ -148,6 +208,23 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
         return http.build();
     }
 
+    // Client Registration Repository for OAUTH2 OIDC Login
+    @Bean
+	public ClientRegistrationRepository clientRegistrationRepository() {
```

Comment:
We are creating the clientRegistration bean even when OAuth is disabled here, do we want to use a spring conditional annotation here?